static/service-worker-registration.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+if ('serviceWorker' in navigator) {
+  window.addEventListener('load', function() {
+    if( typeof __NEXT_DATA__ !== 'undefined' && __NEXT_DATA__.nextExport === true ){
+      navigator.serviceWorker.register('/2.0.0/service-worker.js').then(function(reg) {
+        reg.onupdatefound = function() {
```

Comment:
I think the purpose of this is to tell the user that the page has been made available offline, or that the content has been updated. Google typically implements a brief toast notification to notify the user in their examples, but since that's not a strict requirement we can safely remove it.